### PR TITLE
Create the contributing file

### LINKED ISSUE #12: Adding CONTRIBUTING.md file
Please assign this issue to me with hacktoberfest lable.... Thanks

### PR DESCRIPTION
**Pull request by PravinPadalkar78**
added contributing.md file
* Fix #12

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+
+<br />
+
+### Synchronizing your fork
+
+Open Source projects have a number of contributors who can push code anytime. So it is necessary to make your forked copy equal with the original repository. The remote added above called Upstream helps in this.
+
+`$ git checkout main`
+`$ git fetch upstream`
+`$ git merge upstream/main`
```
Comment (0x61nas) [inline CONTRIBUTING.md]: Our main branch is named as `master` not `main`

**Review by 0x61nas — CHANGES_REQUESTED** [review]
I like it ❤️ 
But you forget to change the links to our repo link instead of `Simple-Library-Managmentissues`

File: CONTRIBUTING.md (hunk 1 above)
Comment (PravinPadalkar78) [reply CONTRIBUTING.md]: sorry for the inconvenience i'll will fix it

Comment (0x61nas) [reply CONTRIBUTING.md]: Not a problem bro

Comment (PravinPadalkar78) [reply CONTRIBUTING.md]: Is It done?

**Review by 0x61nas — CHANGES_REQUESTED** [review]
You still forget some old links, but I can help you if you want

**Comment by PravinPadalkar78** [comment]
Yes please, i would be great 😃 thanks

**Comment by anaselgarhy** [comment]
Ok, I'll make pull request into your fork soon @PravinPadalkar78

**Review by 0x61nas — APPROVED** [review]
Everything is fine, thank you 🤍

**Pull request merged**